src/SA/locale/entry.c

```diff
@@ -0,0 +1,36 @@
+#include <windows.h>
+#include "bofdefs.h"
+#include "base.c"
+#include "anticrash.c"
+
+DECLSPEC_IMPORT int WINAPI KERNEL32$GetLocaleInfoEx(LPCWSTR lpLocaleName, LCTYPE LCType, LPWSTR lpLCData, int cchData);
+WINBASEAPI int WINAPI KERNEL32$GetSystemDefaultLocaleName(LPCWSTR lpLocaleName, int cchLocaleName);
+DECLSPEC_IMPORT LCID WINAPI KERNEL32$LocaleNameToLCID(LPCWSTR lpName, DWORD dwFlags);
+DECLSPEC_IMPORT int WINAPI KERNEL32$GetDateFormatEx(LPCWSTR lpLocaleName, DWORD dwFlags, const SYSTEMTIME *lpData, LPCWSTR lpFormat, LPWSTR lpDateStr, int cchDate, LPCWSTR lpCalendar);
+VOID go( 
+	IN PCHAR Buffer, 
+	IN ULONG Length 
+) 
+{
+
+	int iResult;
+	int BUFFER_SIZE = 512;
```

Comment:
Since this is being used as a define, please change it into one 

ie. above go:
#define BUFFER_SIZE 512

Reply:
Heh, I originally had it that way and thought it was something you wouldn't like. I'll put it back.